evalml/tests/automl_tests/test_automl.py

```diff
@@ -412,6 +413,15 @@ def test_automl_bad_data_check_parameter_type():
         automl.search(data_checks=[MockDataCheckErrorAndWarning])
 
 
+def test_validate_data_check_n_splits(X_y_multi):
```

Comment:
Maybe a better way of testing this is to run `search(data_checks='auto')` on a dataset where one of the classes only has 7 observations.

That way we don't have to call the private `_validate_data_checks` or rely on the fact that the `ClassImbalanceDataCheck` is the last one in the list or that `cv_folds` actually corresponds to `2 * n_splits`.

Reply:
Ah I got it, so you're recommending I validate that the parameter is being passed by triggering the ClassImbalanceDataCheck error, nice!